EnglishFilter/sections/specific.txt

```diff
@@ -294,6 +294,7 @@ linkjust.com,updatesmarugujarat.in,mexashare.com,links4u.me,linku.us,skips.link,
 !
 !
 !
+mangatone.com##.c-sidebar
```

Comment:
I fixed this directly - it's better to use ext-css rules else your rule will block content.